hack/dailybuild/go.mod

```diff
@@ -0,0 +1,5 @@
+module dailybuild
+
+go 1.16
+
+require cloud.google.com/go/storage v1.15.0
```

Comment:
Just an FYI, there is a newer version of the package https://pkg.go.dev/cloud.google.com/go/storage `v1.18.2` as of this writing